templates/usergrams.pug

```diff
@@ -9,8 +9,15 @@ block content
       ul.list-unstyled.mb-0
         li
           h1 #{user.name} | #{user.username}
-        li
-          h3.lead Lover of coffee, composing, and coding.
+        if user.bio
+          li
+            h3.lead #{user.bio}
+        else if currentUser.id === user.id
+          li
+            form(action=`/users/${currentUser.username}/bio` method="post")
+              div(class="form-group")
+                input(class="form-control" type= "text" name="bio" placeholder="Why don't you introduce yourself quickly?" autofocus)
+              button(type="submit" class="btn btn-primary") Post your bio
```

Comment:
Hmm, the user has no way to modify their bio? This is okay for now, but I go ahead and add a story for "modifying my bio" ;)